### PR TITLE
[RFR] Added “V8” entry

### PR DESCRIPTION
**Pull request by bnhovde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -160,7 +160,7 @@ If you are willing to contribute, open a pull request to complete, update or fil
 
 ### V
 
-* [V8](/glossary/V8.md): *to be completed.*
+* [V8](/glossary/V8.md): V8 is Google's open source javaScript engine. It's what Chrome is running, but it's also used for other projects like Node.js and MongoDB.
```
Comment (KittyGiraudel) [inline README.md]: Could we get rid of `V8 is` at the beginning of the sentence?
Also would be super could if we could use smart quotes (`’`) in place of dumb quotes. :)

File: glossary/V8.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # V8
 
-[V8](https://code.google.com/p/v8/) section to be completed.
+[V8](https://code.google.com/p/v8/) is Google's open source, high performance JavaScript engine. It is written in C++ and is used in Google Chrome and other projects.
```
Comment (KittyGiraudel) [inline glossary/V8.md]: If we could use smart quotes here as well! :)

File: README.md (hunk 1 above)
Comment (bnhovde) [inline README.md]: Good idea! 2 secs

**Comment by KittyGiraudel** [comment]
Thanks!

**Pull request merged**